src/In.ProjectEKA.HipService/Common/HttpRequestHelper.cs

```diff
@@ -0,0 +1,40 @@
+namespace In.ProjectEKA.HipService.Common
+{
+    using System;
+    using System.Net.Http;
+    using System.Net.Mime;
+    using System.Text;
+    using Microsoft.Net.Http.Headers;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
+    public static class HttpRequestHelper
+    {
+        public static HttpRequestMessage CreateHttpRequest<T>(
+            string url,
+            T content,
+            string token,
+            string cmSuffix = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5570468)